ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorGroupByOperator.java

```diff
@@ -450,7 +450,7 @@ public void initialize(Configuration hconf) throws HiveException {
                 + "minReductionHashAggr:{} ", maxHtEntries, groupingSets.length,
             numRowsCompareHashAggr, minReductionHashAggr);
       }
-      computeMemoryLimits();
+      computeMemoryLimits(HiveConf.getVar(hconf, HiveConf.ConfVars.HIVE_EXECUTION_ENGINE).equals("tez"));
```

Comment:
can we set `isTez` in initializeOp, similar to `isLlap`?

Reply:
I agree but I am wondering if we should take # of executors into account if it is LLAP. I'd like you to find an LLAP expert if possible...
https://github.com/apache/hive/blob/18f34e75da0141d37d9a8f1cef4f7f64ba09fadb/ql/src/java/org/apache/hadoop/hive/ql/exec/GroupByOperator.java#L875-L877